mycroft/res/text/en-us/limited_stt.voc

```diff
@@ -0,0 +1,10000 @@
+the
```

Comment:
consider a more descriptive name for this `common_words.voc`, `essential_words.voc`?

Reply:
funny, those were my initial names but i found limited_stt more descriptive...

Reply:
`limited_stt` seems ambiguous to me; my first assumption was that it included intent-related vocabulary.. maybe `base_vocabulary`, `basic_vocabulary`, `foundational_words`? I see this as the set of words that a domain-specific vocabulary might be added to (i.e. load a list of medical terms for some medical application or a list of product names for a retail application)